test/test.go

```diff
@@ -17,9 +17,27 @@ limitations under the License.
 package test
 
 import (
+	"flag"
+
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/client-go/kubernetes/scheme"
+
+	"github.com/kubermatic/utils/pkg/testutil"
+
+	"github.com/kubermatic/bulward/pkg/apis/apiserver"
+	apiserverv1alpha1 "github.com/kubermatic/bulward/pkg/apis/apiserver/v1alpha1"
+	corev1alpha1 "github.com/kubermatic/bulward/pkg/apis/core/v1alpha1"
 )
 
 var (
-	testScheme = scheme.Scheme
+	cleanUpStragety = string(testutil.CleanupOnSuccess)
+	testScheme      = scheme.Scheme
 )
+
+func init() {
+	utilruntime.Must(corev1alpha1.AddToScheme(testScheme))
+	utilruntime.Must(apiserver.AddToScheme(testScheme))
+	utilruntime.Must(apiserverv1alpha1.AddToScheme(testScheme))
```

Comment:
why do you need both `apiserver` and `apiserverv1alpha1`?

Reply:
done